developer/developer-faqs.md

```diff
@@ -3,6 +3,16 @@
 ## Installation
 
 ## General
+Q: What do I need to know to work with Reaction?
+
+A: To start hacking Reaction, you should probably have a basic understanding of the following technologies:
+- JavaScript, specifically ES6
+- HTML/CSS
+- Meteor experience would definitively help
```

Comment:
I think something like "Some knowledge of Meteor, especially an understanding of [Publications and Subscriptions](https://guide.meteor.com/data-loading.html)"

Reply:
👍

Reply:
pushed.